internal/mill/exec_test.go

```diff
@@ -384,14 +384,15 @@ func newPetFixture(t *testing.T) *petFixture {
 	dir, _ := ioutil.TempDir("", t.Name())
 	wmDir := dirs.NewWindmillDirAt(dir)
 	procfs, err := proc.NewProcFSWithDir(wmDir)
+	drymode := false
 	if err != nil {
 		t.Fatal(err)
 	}
 	runner := proc.NewRunner(procfs)
 	school := school.NewPetSchool(procfs)
 	return &petFixture{
 		t:         t,
-		petsitter: NewPetsitter(stdout, stderr, runner, procfs, school),
+		petsitter: NewPetsitter(stdout, stderr, runner, procfs, school, drymode),
```

Comment:
it might be worth having a test with dryrun on and checking the output!

Reply:
Dry run test for run is up! I could add more. And I don't know if my solution for a new pet fixture is all that elegant?
I think they CI will fail because I don't have the health checks or something?